### DISCUSSION TITLE
I don't understand Korean

### DISCUSSION THREAD

**Discussion by Hyunmin0502** [mmuhunn/CampusShare#52] [General]
On Top contributor Ranking section, there is this Korean 기여: 200회. And I don't understand this. Anyone like me would not understand this.

**Comment by mmuhunn** [comment]
I don't know where. Can you show where it is by adding screenshot?😅😅

**Comment by Hyunmin0502** [comment]
<img width="512" alt="화면 캡처 2024-12-16 232554" src="https://github.com/user-attachments/assets/d2212547-7252-4f20-8c76-87d53aa080e9" />
This one

**Comment by Hyunmin0502** [comment]
Okay I will fix myself 😡
